src/service/viewer-impl.js

```diff
@@ -790,15 +791,26 @@ export class Viewer {
   }
 
   /**
-   * Retrieves the Base CID from the viewer
+   * Get/set the Base CID from/to the viewer.
+   * @param {string=} opt_data Stringified JSON object {cid, time}.
    * @return {!Promise<string|undefined>}
    */
-  getBaseCid() {
+  baseCid(opt_data) {
```

Comment:
Do we have to change the name here? We can get caught into version skew if we do. Another option is to leave the old name and deprecate and remove in a week or two.